config/tslint.json

```diff
@@ -1,30 +1,34 @@
 {
-  "extends": [
-    "tslint:recommended",
-    "tslint-config-prettier"
-  ],
-  "rules": {
-    "jsdoc-format": true,
-    "object-literal-sort-keys": false,
-    "only-arrow-functions": [false],
-    "max-classes-per-file": [false],
-    "max-line-length": [180],
-    "member-access": false,
-    "member-ordering": [false],
-    "no-shadowed-variable": false,
-    "no-var-requires": true,
-    "no-angle-bracket-type-assertion": false,
-    "no-console": [false],
-    "no-empty-interface": false,
-    "no-string-literal": false,
-    "interface-name": [false],
-    "interface-over-type-literal": false,
-    "variable-name": [
-      true,
-      "ban-keywords",
-      "check-format",
-      "allow-leading-underscore",
-      "allow-pascal-case"
-    ]
-  }
+	"extends": "tslint:recommended",
+	"rules": {
+		"indent": [
+			true,
+			"tabs"
+		],
```

Comment:
This can potentially cause problems when used in conjunction with prettier, as prettier sometimes does some indenting with spaces (simply for formatting object fields containing ORs) so if we could solve these two modules fighting that would be nice. An example:
```typescript
				target != null
					? image.dockerImageId === target.config.image ||
					  image.name === target.imageName
					: false,
```

Notice the two spaces aligning `image.dockerImageId` and `image.name`

Reply:
Yup, found it today and working on a fix. Extending tslint with `"tslint-config-prettier"` when prettier option is enabled works as it disables all formatting rules.